src/main/java/org/takes/facets/auth/codecs/CcXOR.java

```diff
@@ -63,7 +63,13 @@ public CcXOR(final Codec codec, final String key) {
      * Ctor.
      * @param codec Original codec
      * @param key Secret key for encoding
+     * @todo #558:30min CcXOR ctor. According to new qulice version,
+     *  constructor must contain only variables initialization and other
+     *  constructor calls. Refactor code according to that rule and remove
+     *  `ConstructorOnlyInitializesOrCallOtherConstructors`
+     *  warning suppression.
      */
+    @SuppressWarnings("PMD.ConstructorOnlyInitializesOrCallOtherConstructors")
```

Comment:
@xupyprmv please fix that constructor instead of creating puzzle, it looks simple